tendenci/apps/events/forms.py

```diff
@@ -2127,13 +2132,13 @@ def __init__(self, registrant, is_admin, *args, **kwargs):
 
         for index, start_dt in enumerate(sub_event_datetimes.keys()):
             child_events = upcoming_child_events.filter(start_dt=start_dt)
-            choices = [(event.pk, f'{event.event_code} - {event.title}' if event.event_code else event.title) for event in child_events if not event.at_capacity]
+            choices = [(event.pk, event.title_with_event_code) for event in child_events if not event.at_capacity]
             # Check if registrant already has selection. If so, make sure it's in choices
```

Comment:
Property added in next commit. This is so that we don't have duplicate options when an event is already selected